nextjs-13/src/app/page.tsx

```diff
@@ -28,6 +44,12 @@ const Home = () => {
         <div className="home__tab">Newsletter</div>
       </div>
       <div className="home__dashboard">
+        <div>
+          <h3>Your Newsletters</h3>
+          <Suspense fallback={<ClipLoader loading={true} size={150} />}>
```

Comment:
That loader is quite massive 🤣 We could probably decrease its size a bit

Reply:
Resolved [here](https://github.com/nimblehq/js-framework-benchmark/pull/110/commits/d30f9f526aeaccaa44ac7bfcc866291c1952f753#diff-223e9b714cf2c9b5f163079f0bd63d3a6eac9c46915fc3afde20a2742897d706R49).